models/Document/Editable.php

```diff
@@ -325,18 +325,14 @@ public function setConfig(array $config): static
      */
     public function addConfig(string $name, mixed $value): static
     {
-        if (!is_array($this->config)) {
-            $this->config = [];
-        }
-
         $this->config[$name] = $value;
 
         return $this;
     }
 
-    public function getRealName(): string
+    public function getRealName(): ?string
```

Comment:
Don't change the return type. If realName is `null` it will return `''` now.
```suggestion
    public function getRealName(): string
```